server.test.js

```diff
@@ -0,0 +1,40 @@
+const request = require('supertest');
+const server = require('./server');
+
```

Comment:
In this file you can create an object that represents your database data. Then this object can be used to write more tests. For examples: 
`const games = [
  { id: 1, title: "Pacman", genre: "Arcade", releaseYear: 1980 },
  { id: 2, title: "Fortnite", genre: "Battle Royale", releaseYear: 2017 },
  { id: 3, title: "Assassins Creed", genre: "Action", releaseYear: 2017 },
  { id: 4, title: "Mario Cart", genre: "Kids Wii Gaming", releaseYear: 2000 }
];`